src/Microsoft.ServiceFabric.Services.Wcf/Remoting/V2/Wcf/Client/WcfServiceRemotingClient.cs

```diff
@@ -3,30 +3,37 @@
 // Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 // ------------------------------------------------------------
 
+using System;
+using System.Collections.Generic;
+using System.Fabric;
+using System.ServiceModel;
+using System.Threading.Tasks;
+

```

Comment:
Nit: I don't think micro-managing the using blocks this is useful. The "Remove and Sort Usings" refactoring does a good job, we just need to make sure the option to place System namespaces first is enabled. This is the old default that changed a few years ago, but once you enable it VS/Code should remember it in your cloud settings.